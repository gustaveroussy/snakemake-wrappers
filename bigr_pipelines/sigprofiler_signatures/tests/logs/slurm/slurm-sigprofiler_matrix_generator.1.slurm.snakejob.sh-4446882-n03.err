localrules directive specifies rules that are not present in the Snakefile:
	bigr_copy

Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Singularity containers: ignored
Select jobs to execute...

[Thu Dec 16 16:17:35 2021]
Job 0: Building substitution matrices with SigProfiler on s050

mkdir --parents --verbose SigProfiler/s050/test/input/input > logs/sigprofiler/matrix_generator/s050.log 2>&1 && gzip --stdout --decompress --force --verbose snpsift/fixed/s050.vcf.gz > SigProfiler/s050/test/input/s050.vcf 2>> logs/sigprofiler/matrix_generator/s050.log && python3 /mnt/beegfs/pipelines/snakemake-wrappers/bigr_pipelines/sigprofiler_signatures/scripts/matrix_generator.py snpsift/fixed/s050.vcf.gz SigProfiler/s050/s050_matrix.yaml --organism GRCh38 >> logs/sigprofiler/matrix_generator/s050.log 2>&1 
Activating conda environment: /mnt/beegfs/pipelines/snakemake-wrappers/bigr_pipelines/conda/7128fb4d1f0c93e2e437943da1e47ea9
[Thu Dec 16 16:17:38 2021]
Error in rule sigprofiler_matrix_generator:
    jobid: 0
    output: SigProfiler/s050/s050_matrix.yaml, SigProfiler/s050/test/input, SigProfiler/s050/test/logs, SigProfiler/s050/test/output
    log: logs/sigprofiler/matrix_generator/s050.log (check log file(s) for error message)
    conda-env: /mnt/beegfs/pipelines/snakemake-wrappers/bigr_pipelines/conda/7128fb4d1f0c93e2e437943da1e47ea9
    shell:
        mkdir --parents --verbose SigProfiler/s050/test/input/input > logs/sigprofiler/matrix_generator/s050.log 2>&1 && gzip --stdout --decompress --force --verbose snpsift/fixed/s050.vcf.gz > SigProfiler/s050/test/input/s050.vcf 2>> logs/sigprofiler/matrix_generator/s050.log && python3 /mnt/beegfs/pipelines/snakemake-wrappers/bigr_pipelines/sigprofiler_signatures/scripts/matrix_generator.py snpsift/fixed/s050.vcf.gz SigProfiler/s050/s050_matrix.yaml --organism GRCh38 >> logs/sigprofiler/matrix_generator/s050.log 2>&1 
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Removing output files of failed job sigprofiler_matrix_generator since they might be corrupted:
SigProfiler/s050/test/input
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
