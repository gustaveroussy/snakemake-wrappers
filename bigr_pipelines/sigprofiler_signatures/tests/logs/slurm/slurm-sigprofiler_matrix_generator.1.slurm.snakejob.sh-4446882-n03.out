n03
Thu Dec 16 16:17:33 CET 2021
